src/cf-forms/src/organisms/form.less

```diff
@@ -10,7 +10,7 @@
         padding: 0;
 
         .m-field + .m-field {
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/528/commits/9f7e7bd4d6498c2ae581a3e5e1efe5dd07be7d84